readme.txt

```diff
@@ -1,16 +1,16 @@
 === Doliwoo ===
 Contributors: GPC.solutions
-Donate Link: TODO (https://gpcsolutions.fr/donate)
+Donate Link: TODO (https://gpcsolutions.fr/donate) > 404 PAGE
 Tags: dolibarr, woocommerce, ecommerce, erp, integration
 Author URI: https://gpcsolutions.fr
-Plugin URI: TODO (https://gpcsolutions.fr/doliwoo)
-Requires at least: 3.7.1
+Plugin URI: TODO (https://gpcsolutions.fr/doliwoo) > 404 PAGE 
+Requires at least: 3.7.1 (Dolibarr)
```

Comment:
No this is the standard WordPress readme so this refers to the WordPress version.